supportedBackends/sfs/src/main/scala/cromwell/backend/impl/sfs/config/MemoryDeclarationValidation.scala

```diff
@@ -83,9 +85,13 @@ class MemoryDeclarationValidation(declaration: Declaration, attributeName: Strin
   }
 
   private def coerceMemorySize(womType: WomType)(value: MemorySize): WomValue = {
+    val amount = value.to(declarationMemoryUnit).amount
     womType match {
-      case WomIntegerType => WomInteger(value.to(declarationMemoryUnit).amount.toInt)
-      case WomFloatType => WomFloat(value.to(declarationMemoryUnit).amount)
+      case WomIntegerType if amount.inIntRange => WomInteger(amount.toInt)
+      // Should we fail here instead ?
```

Comment:
ToL: Can this be answered definitively before merging? Or, assuming WDL Int is going to be expanded, this comment could mention a github ticket/issue for followup? It looks like the previous version just failed.